rules/windows/command_and_control_dns_tunneling_nslookup.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/11/11"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of Nslookup process execution with explicit query type from the same host. This may indicate a
```

Comment:
```suggestion
This rule identifies a large number of Nslookup process executions with an explicit query type from the same host. This may indicate Command and Control activity via the DNS Protocol.
```